tempesta_fw/sock.c

```diff
@@ -114,14 +114,25 @@ ss_skb_entail(struct sock *sk, struct sk_buff *skb)
 	tcb->seq     = tcb->end_seq = tp->write_seq;
 	tcb->tcp_flags = TCPHDR_ACK;
 	tcb->sacked  = 0;
-	skb_header_release(skb);
+	__skb_header_release(skb);
```

Comment:
`ss_skb_unroll` makes `skb` private for us. If it were referenced by someone except us, then private copy of SKB chain will be allocated. So, in my opinion, it's no reason to use `skb_header_release` version here anymore.